cbw_api_toolbox/cbw_api.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from json import JSONDecodeError
+from typing import List
+
+import requests
+from requests import Response
+
+from cbw_api_toolbox.__routes__ import ROUTE_GET_SERVERS
+from cbw_api_toolbox.cbw_auth import CBWAuth
+from cbw_api_toolbox.cbw_objects.cbw_server import CBWServer
+from cbw_api_toolbox.cbw_parser import CBWParser
+
+
```

Comment:
remove this line (double space)

Reply:
voir https://www.python.org/dev/peps/pep-0008/#blank-lines